### ISSUE TITLE
`assh config list` shows odd value for ServerAliveInterval

### ISSUE THREAD

**Issue by ocean90** [moul/advanced-ssh-config#181]
```
➜  ~ assh -v
assh version 2.4.1+dev (HEAD)

➜  ~ assh config build | grep ServerAliveInterval
  ServerAliveInterval 60

➜  ~ assh config list | grep ServerAliveInterval
        ServerAliveInterval: <
```

Any idea where the `<` is coming from?

**Comment by moul** [comment]
Ahah, fun issue,

“<“ is the ascii char for the decimal 60 value

I will fix it, thanks for the report

**Comment by moul** [comment]
Fixed by #193, can you check again?

**Comment by ocean90** [comment]
@moul Works like a charm, thanks!
